CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+# Contributing
+
+Thanks for helping with SlimeVR! This document explains what to know when
+contributing.
+
+We are most active on [discord](https://discord.gg/SlimeVR), come chat with us if you
+have any questions or want to hang out! 😎
+
+
+## Setting up PlatformIO
+
+We use platformio as our build tooling. To set this up, we recommend using the
+[VSCode](https://code.visualstudio.com/) editor, and installing the platformio
+extension:
+![](https://docs.platformio.org/en/latest//_images/platformio-ide-vscode-pkg-installer.png)
+
+There is also a [command line version](https://docs.platformio.org/en/latest//core/index.html#piocore)
+of platformio if you prefer that, that you can use instead.
+
+
+## Using third-party code
+
+Often you will want to use or reference third party code. You should not just
+copy-paste code from random libraries! There are several reasons for this:
+
+- It is not clear when/where the code originated from.
+- The original code's license needs to be kept intact.
+- It removes the ability to update our code when the original updates.
+- It makes it harder for us to maintain the code, because we don't what has changed
+  from the original!
+
+To bring a new library into the firmware, we instead treat the modified library as a
+"fork" of the original. We then add the forked library as a dependency of the firmware,
+via platformio which is our build tool.
+
+
+### Forking
+
+> **Note**
+> If you have never forked in git before, a fork is basically a copy of a library that
+> maintains the original's history. This means that any changes you commit in your fork
+> will show up as changes against the original. We can even update these changes as the
+> original changes!
+
+To create a fork in github, simply go to the original library and click this button:
+![](https://docs.github.com/assets/cb-28613/images/help/repository/fork_button.png)
+Now clone your repo to your computer. From this point forward, any changes that you
+make while working on your fork will have their git history preserved and linked to the
+original.
+
+### Adding the fork as a dependency
```

Comment:
I just not used to fork... I think one of the below is more easy to understand:
- \### Adding a third party library as a dependency 
- \### How to integrate a third party library

Reply:
What do you think about the latest changes?

Reply:
Looks good